packages/mui-base/src/Unstable_NumberInput/NumberInput.test.tsx

```diff
@@ -2,13 +2,23 @@ import * as React from 'react';
 import { expect } from 'chai';
 import { spy } from 'sinon';
 import userEvent from '@testing-library/user-event';
-import { act, createMount, createRenderer, describeConformanceUnstyled } from 'test/utils';
+import {
+  act,
+  createMount,
+  createRenderer,
+  describeConformanceUnstyled,
+  fireEvent,
+} from 'test/utils';
 import {
   Unstable_NumberInput as NumberInput,
   numberInputClasses,
   NumberInputOwnerState,
 } from '@mui/base/Unstable_NumberInput';
 
+// TODO for v6 initialize @testing-library/user-event using userEvent.setup() instead of directly calling methods e.g. userEvent.click() for all related tests in this file
```

Comment:
Maybe consider using `TODO v6:` phrase instead, as it's consistently used throughout the entire codebase and can be easily found in a global search.
```suggestion
// TODO v6: Initialize @testing-library/user-event using userEvent.setup() instead of directly calling methods e.g. userEvent.click() for all related tests in this file
```